core/src/net/sf/openrocket/aerodynamics/barrowman/FinSetCalc.java

```diff
@@ -645,7 +645,7 @@ public double calculatePressureDragForce(FlightConditions conditions,
 		// Airfoil assumed to have zero base drag
 		
 		// Scale to correct reference area
-		drag *= finCount * span * thickness / conditions.getRefArea();
+		drag *= span * thickness / conditions.getRefArea();
```

Comment:
good catch here - it took me a few minutes to realize why the finCount was removed here (due to using the activeInstances rather than activeComponents)

Reply:
:)